libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -724,14 +748,8 @@ - (void) finishLoadActions:(UIView *) webView navigation:(WKNavigation *) naviga
                 _foundHomeUrl = YES;
             }
         }
-        if (self.useUIWebView) {
-            [CDVUserAgentUtil releaseLock:self.userAgentLockToken];
-            [[NSNotificationCenter defaultCenter] postNotification:[NSNotification notificationWithName:CDVPageDidLoadNotification object:self.webView]];
-        } else {
-            if (self.navWebViewDelegate) {
-                [self.navWebViewDelegate webView:(WKWebView *) webView didFinishNavigation:navigation];
-            }
-        }
+        [CDVUserAgentUtil releaseLock:self.userAgentLockToken];
```

Comment:
Same behavior for both `WKWebView` and `UIWebView` here since they both use the stock plugins now.